dev-python/django-tables2/django-tables2-2.4.1.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..10} )
+
+inherit distutils-r1
+
+DESCRIPTION="Table/data-grid framework for Django"
+HOMEPAGE="https://pypi.org/project/django-tables2/ https://github.com/bradleyayers/django-tables2/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="BSD-2"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND=">=dev-python/django-1.11"
+DEPEND="${RDEPEND}"
```

Comment:
`DEPEND` is rarely correct in Python packages. https://projects.gentoo.org/python/guide/basic.html#dependencies-in-python-packages

Reply:
Also enable tests, please.